Library/Homebrew/github_packages.rb

```diff
@@ -38,35 +39,131 @@ def upload_bottles(bottles_hash)
     raise UsageError, "HOMEBREW_GITHUB_PACKAGES_USER is unset." if user.blank?
     raise UsageError, "HOMEBREW_GITHUB_PACKAGES_TOKEN is unset." if token.blank?
 
-    oras   = Formula["oras"].opt_bin/"oras" if Formula["oras"].any_version_installed?
-    oras ||= begin
+    docker = HOMEBREW_PREFIX/"bin/docker"
+    unless docker.exist?
+      ohai "Installing `docker` for upload..."
+      safe_system HOMEBREW_BREW_FILE, "install", "--formula", "docker"
```

Comment:
I personally use the Docker Desktop for macOS cask. The Docker client formula is more lightweight is sufficient for this purpose. No change needed, because `HOMEBREW_PREFIX/"bin/docker"` can be satisfied by either formula or cask.

Reply:
This code will be run by CI on Linux, where it'd be faster to use the Docker client provided by GitHub Actions image if done outside of the `Homebrew/brew` Docker image. Doesn't need to be done in this PR, just an idea for a future refinement.

Reply:
> No change needed, because `HOMEBREW_PREFIX/"bin/docker"` can be satisfied by either formula or cask.

Yes, this is why I made this code do what it did.

> This code will be run by CI on Linux, where it'd be faster to use the Docker client provided by GitHub Actions image if done outside of the `Homebrew/brew` Docker image. Doesn't need to be done in this PR, just an idea for a future refinement.

Yeh, makes sense. Not something I'll be getting to personally, though.